packages/fxa-settings/src/lib/cache.ts

```diff
@@ -77,6 +77,20 @@ export function sessionToken(newToken?: hexstring) {
   }
 }
 
+export function discardSessionToken() {
```

Comment:
I would probably add a comment here that this function is intended to discard an invalid session. It could be read as destroying a session otherwise.

Reply:
Thanks, I added a comment.

In content-server, this runs `this.unset('sessionToken')` so it takes the session token off the account model. It seemed right that in React we should just set it to `undefined`.